ess/src/API/EMBC.ESS/Resources/Evacuations/EvacuationRepository.cs

```diff
@@ -192,7 +198,7 @@ private static void AssignHouseholdMember(EssContext essContext, era_evacuationf
             if (registrant == null) throw new ArgumentException($"Household member has registrant id {member._era_registrant_value} which was not found");
             essContext.AddLink(registrant, nameof(contact.era_contact_era_householdmember_Registrantid), member);
         }
-        essContext.AddLink(file, nameof(era_evacuationfile.era_era_evacuationfile_era_householdmember_EvacuationFileid), member);
+        //essContext.AddLink(file, nameof(era_evacuationfile.era_era_evacuationfile_era_householdmember_EvacuationFileid), member);
```

Comment:
do you want to remove this line and similar commented line about pets in next function

Reply:
removed